cmd/dagger/cmd/common/common.go

```diff
@@ -82,6 +84,21 @@ func ValueDocOneLine(val *compiler.Value) string {
 	return strings.Join(docs, " ")
 }
 
+func convertRelativePaths(cacheOptionEntries []buildkit.CacheOptionsEntry) []buildkit.CacheOptionsEntry {
+	var convertedOptions = cacheOptionEntries
```

Comment:
If this variable necessary?

Reply:
No 👍🏻 Initially I was wondering if adding a copy/clone of the array would be beneficial. (To prevent mutation of passed params). But I guess here it doesn't really make a big difference. (This was a placeholder ^^).